jax/experimental/jax2tf/README.md

```diff
@@ -708,6 +708,81 @@ g_jaxx2tf_0 = tape.gradient(res, xs,
 # In this case we get the same result as for TF native.
 ```
 
+### Functions whose arguments and results are Python nested data structures
+
+jax2tf can convert functions with arguments and results that are nested
+collections (tuples, lists, dictionaries) of numeric values or JAX arrays. The
```

Comment:
Do you want to just reference to PyTree definition? https://jax.readthedocs.io/en/latest/pytrees.html, or there are limitations to support generic PyTrees?

Reply:
Done